pkgs/tools/misc/renameutils/default.nix

```diff
@@ -9,9 +9,25 @@ stdenv.mkDerivation rec {
     sha256 = "18xlkr56jdyajjihcmfqlyyanzyiqqlzbhrm6695mkvw081g1lnb";
   };
 
-  patches = [ ./install-exec.patch ];
+  patches = [ ./install-exec.patch ] ++ lib.optionals stdenv.isDarwin [ ./darwin.patch ];
```

Comment:
Optional patches are not great. Also we do not use the g prefix for coreutils normally. Please try to wrap instead. Also why do we need to patch this first an later replace?

Reply:
I have removed the patch in favour of doing the substitutions directly in the postpatch substituteInPlace.